common/lib/xmodule/xmodule/templates/problem/jsinput_response.yaml

```diff
@@ -5,67 +5,176 @@ metadata:
+            message = str(int(round(final_grade, 2) * 100)) + '% overlap with correct answer.'
+
+            if interval_tolerance == 'strict':
+              final_grade = final_grade * final_grade
+            elif interval_tolerance == 'generous':
+              final_grade = math.sqrt(final_grade)
+            
+            # Round up to the nearest tenth.
+            final_grade = math.ceil(final_grade*10.0) / 10.0
+            
+            if show_open_close:
+              if(guess_lower_closed != True and interval_type[0] == 'closed'):
+                final_grade = final_grade - type_penalty
+                message += ' Lower endpoint is wrong.'
+              if(guess_lower_closed == True and interval_type[0] != 'closed'):
+                final_grade = final_grade - type_penalty
+                message += ' Lower endpoint is wrong.'
+              if(guess_upper_closed != True and interval_type[1] == 'closed'):
+                final_grade = final_grade - type_penalty
+                message += ' Upper endpoint is wrong.'
+              if(guess_upper_closed == True and interval_type[1] != 'closed'):
+                final_grade = final_grade - type_penalty
+                message += ' Upper endpoint is wrong.'
+            
+        else:
+          
+          farthest = max(abs(correct_number - guess_upper), abs(correct_number - guess_lower))
+          
+          if farthest &lt; tolerance[0]:
+            final_grade = brackets[0]
+            message = 'Close enough! Actual answer: ' + str(correct_number)
+          elif farthest &lt; tolerance[1]:
+            final_grade = brackets[1]
+            message = 'Close. You are off by ' + str(farthest)
+          elif farthest &lt; tolerance[2]:
+            final_grade = brackets[2]
+            message = 'Not very close. You are off by ' + str(farthest)
+          else:
+            final_grade = brackets[3]
+            message = 'Your range is too large to get points.'
+          
+          if guess_upper &gt; correct_number and guess_lower &lt; correct_number:
+            message += ' The answer is within your range.'
+          else:
+            message += ' The answer is outside your range.'
+          
+        if not feedback:
+          message = ''
+
+        if final_grade &gt; 0.95:
+          isOK = True
+        elif final_grade &gt; 0.05:
+          isOK = "Partial"
+        else:
+          isOK = False
+        
+        return {
+          'input_list': [
+              { 'ok': isOK, 'msg': message, 'grade_decimal': final_grade},
+          ]
+        }
+
+
+
+      </script>
+        <script type="text/javascript">
+      $(document).ready(function(){
+      	console.log('Let\'s go!');
+      });
+
+      // Logs a javascript object.
+      function logThatThing(ThatThing){
+
+      	// Log it to the console just to verify it's working
+      	console.log(JSON.stringify(ThatThing));
+      	
+      	// Send it to the official edX logamajig!
```

Comment:
I don't think we want this sort of thing in an example we are putting out. Can we ditch the whole "logThatThing"?